cmd/dep/status.go

```diff
@@ -320,6 +322,11 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 		params.TraceLogger = ctx.Err
 	}
 
+	logger := ctx.Err
+	if !ctx.Verbose {
+		logger = log.New(ioutil.Discard, "", 0)
+	}
```

Comment:
Can this either be combined with the preceding `if ctx.Verbose` block into an if/else, or can it be moved down closer to it's usage or in a narrower scope?